static/docs/user-guide/contributing-documentation.md

```diff
@@ -12,10 +12,10 @@ To contribute documentation you need to know these locations:
 - [Content](https://github.com/iterative/dvc.org/tree/master/static/docs)
   (`/static/docs`) -
   [Markdown](https://guides.github.com/features/mastering-markdown/) files of
-  the different pages to render dynamically in the browser;
+  the different pages to render dynamically in the browser
 - [Images](https://github.com/iterative/dvc.org/tree/master/static/img)
   (`/static/img`) - add new images, gif, svg, etc here. Reference them from the
-  Markdown files like this: `![](/static/img/reproducibility.png)`;
+  Markdown files like this: `![](/static/img/reproducibility.png)`
```

Comment:
Please change `) - add` for `): Add` in the beginning of this bullet, and end it with period `.`.